src/Data/Number.purs

```diff
@@ -53,3 +113,251 @@ fromString :: String -> Maybe Number
 fromString str = runFn4 fromStringImpl str isFinite Just Nothing
 
 foreign import fromStringImpl :: Fn4 String (Number -> Boolean) (forall a. a -> Maybe a) (forall a. Maybe a) (Maybe Number)
+
+-- | Returns the absolute value of the argument.
+-- | ```purs
+-- | > x = -42.0
+-- | > sign x * abs x == x
+-- | true
+-- | ```
+foreign import abs :: Number -> Number
+
+-- | Returns the inverse cosine in radians of the argument.
+-- | ```purs
+-- | > acos 0.0 == pi / 2.0
+-- | true
+-- | ```
+foreign import acos :: Number -> Number
+
+-- | Returns the inverse sine in radians of the argument.
+-- | ```purs
+-- | > asin 1.0 == pi / 2.0
+-- | true
+-- | ```
+foreign import asin :: Number -> Number
+
+-- | Returns the inverse tangent in radians of the argument.
+-- | ```purs
+-- | > atan 1.0 == pi / 4.0
+-- | true
+-- | ```
+foreign import atan :: Number -> Number
+
+-- | Four-quadrant tangent inverse. Given the arguments `y` and `x`, returns
+-- | the inverse tangent of `y / x`, where the signs of both arguments are used
+-- | to determine the sign of the result.
+-- | If the first argument is negative, the result will be negative.
+-- | The result is the angle between the positive x axis and a point `(x, y)`.
+-- | ```purs
+-- | > atan2 0.0 1.0
+-- | 0.0
+-- | > atan2 1.0 0.0 == pi / 2.0
+-- | true
+-- | ```
+foreign import atan2 :: Number -> Number -> Number
+
+-- | Returns the smallest integer not smaller than the argument.
+-- | ```purs
+-- | > ceil 1.5
+-- | 2.0
+-- | ```
+foreign import ceil :: Number -> Number
+
+-- | Returns the cosine of the argument, where the argument is in radians.
+-- | ```purs
+-- | > cos (pi / 4.0) == sqrt2 / 2.0
+-- | true
+-- | ```
+foreign import cos :: Number -> Number
+
+-- | Returns `e` exponentiated to the power of the argument.
+-- | ```purs
+-- | > exp 0.0
+-- | 1.0
+-- | ```
+foreign import exp :: Number -> Number
+
+-- | Returns the largest integer not larger than the argument.
+-- | ```purs
+-- | > floor 1.5
+-- | 1.0
+-- | ```
+foreign import floor :: Number -> Number
+
+-- | Returns the natural logarithm of a number.
+-- | ```purs
+-- | > log 1.0
```

Comment:
Same here - maybe `log e` would be more illustrative here? I realise you have that same example on the documentation for `e` but I think repetition here is fine, especially as people might be linked directly to the documentation for `exp` and `log` and not see `e`.

Reply:
Fixed in latest commit.